core/domain/services/graphql/mutators/BulkEntityDelete.php

```diff
@@ -58,6 +59,8 @@ public static function mutateAndGetPayload()
                 $deletionMethod .= '\DatetimeDelete::' . ($deletePermanently ? 'deleteDatetimeAndRelations' : 'trashDatetimeAndRelations');
             } elseif ($details['entityType'] === EEM_Ticket::instance()->item_name()) {
                 $deletionMethod .= '\TicketDelete::' . ($deletePermanently ? 'deleteTicketAndRelations' : 'trashTicket');
+            } elseif ($details['entityType'] === EEM_Price::instance()->item_name()) {
+                $deletionMethod .= '\PriceDelete::deletePriceAndRelations';
```

Comment:
ideally, these classes from the root namespace (those that start with `\`) should be imported into the class, ie:

```php
//top of file:
use DatetimeDelete;
use EEM_Datetime;
use EEM_Ticket;
use EEM_Price;
use EE_Error;
use Exception;
use InvalidArgumentException;
use PriceDelete;
use TicketDelete;

// code

if ($details['entityType'] === EEM_Datetime::instance()->item_name()) {
	$deletionMethod .= 'DatetimeDelete::' . ($deletePermanently ? 'deleteDatetimeAndRelations' : 'trashDatetimeAndRelations');
} elseif ($details['entityType'] === EEM_Ticket::instance()->item_name()) {
	$deletionMethod .= 'TicketDelete::' . ($deletePermanently ? 'deleteTicketAndRelations' : 'trashTicket');
} elseif ($details['entityType'] === EEM_Price::instance()->item_name()) {
	$deletionMethod .= 'PriceDelete::deletePriceAndRelations';
```

but it's not critical